bin/test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+let Promise = require('bluebird');
+let spawn = require('child_process').spawn;
+let fs = Promise.promisifyAll(require('fs'));
+let path = require('path');
+
+let cmd = path.resolve(__dirname, '..', 'node_modules', '.bin', '_mocha');
+let args = ['--compilers', 'ts:ts-node/register,tsx:ts-node/register'];
+
+let services = path.resolve('services');
+return fs.readdirAsync(services).then(folders => {
+  return Promise.each(folders, f => {
+    let dir = path.resolve(services, f);
+    return fs
+      .readdirAsync(dir)
+      .then(subfolders => {
+        if (subfolders.indexOf('test') > 0) {
+          return new Promise((resolve, reject) => {
+            console.log('RUN TESTS - %s:', f);
+            let testArgs = args.push(path.resolve(dir, 'test/**/*test.ts'));
+            let test = spawn(cmd, args);
+            test.stdout.on('data', out => {
+              console.log(out.toString());
+            });
+
+            test.stderr.on('data', out => {
+              console.log(out.toString());
```

Comment:
Can we log like `console.log("Error: " + out.toString());` to distinguish between the stdout and stderr?